modal/partial_function.py

```diff
@@ -53,13 +53,24 @@ def __init__(
         self.wrapped = False  # Make sure that this was converted into a FunctionHandle
 
     def __get__(self, obj, objtype=None) -> _Function:
-        # This only happens inside user methods when they refer to other methods
         k = self.raw_f.__name__
         if obj:  # Cls().fun
-            function = getattr(obj, "_modal_functions")[k]
-        else:  # Cls.fun
-            function = getattr(objtype, "_modal_functions")[k]
-        return function
+            if hasattr(obj, "_modal_functions"):
+                # This happens inside "local" user methods when they refer to other methods,
+                # e.g. Foo().parent_method() doing self.local.other_method()
+                return getattr(obj, "_modal_functions")[k]
+            else:
+                # special edge case: referencing a method of an instance of an
+                # unwrapped class (not using app.cls()) with @methods
+                # not sure what would be useful here, but lets return a bound version of the underlying function,
+                # since the class is just a vanilla class at this point
+                return self.raw_f.__get__(obj, objtype)
+
+        else:  # Cls.fun6
```

Comment:
What does `.fun6` mean?

Reply:
That I have fat fingers :D Supposed to be `Cls.fun` and indicate that this branch covers accessing a method directly on the class rather than via an instance of the class